common/src/main/java/net/william278/huskhomes/command/RtpCommand.java

```diff
@@ -29,18 +29,16 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
 
 public class RtpCommand extends Command implements UserListTabProvider {
 
     protected RtpCommand(@NotNull HuskHomes plugin) {
         super("rtp", List.of(), "[player] [world]", plugin);
-        addAdditionalPermissions(Map.of(
-                "other", true,
-                "world", true
-        ));
+        Map<String, Boolean> map = new HashMap<>();
+        map.put("other", true);
+        plugin.getWorlds().forEach(world -> map.put(world.getName(), true));
```

Comment:
Can we filter this by `plugin.getSettings().getRtp().isWorldRtpRestricted(world)` (so as to not register unneccessary perms)?